examples/lit-components.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Degu Image</title>
+  <style>
+    degu-image {
+       display: inline-block;
+       line-height: 0;
+        width: 100%;
+    }
+
+    degu-image img {
+      max-width: 100%;
+      height: auto;
+    }
+
+    degu-video {
+      display: inline-block;
+      line-height: 0;
+      width: 100%;
+    }
+    video {
+        width: 100%;
+    }
+
+    .section {
+      padding: 20px;
+      max-width: 800px;
+      margin: 0 auto;
+    }
+    .section__header {
+      font-size: 20px;
+      margin-bottom: 20px;
+    }
+
+  </style>
+</head>
+<body init-sample="litComponentSample">
+
+
+  <div class="section">
+    <div class="section__header">Degu Image</div>
+    <degu-image src="https://placeholder-dot-madebygoog.appspot.com/2650x720.svg" alt="Logo" class="header__logo__image"></degu-image>
+    <degu-image
+      width="2650" height="720"
+      src="https://placeholder-dot-madebygoog.appspot.com/2650x720.svg" alt="Logo" class="header__logo__image"></degu-image>
+
+    <degu-image
+      width="2300" height="958"
+      src="https://lh3.googleusercontent.com/EtrsUA5VUfcBjJZqcgfXBLQfvE9lkYMnaU9L8Zp_Y0MzIciD3NDXhYkakb_59Bk8oSCHMfpsDXGJtzrtY9aDYZT3xFic11kBgwM" alt="Logo" class="header__logo__image"></degu-image>
```

Comment:
Use placeholder assets not tied to a specific client project

Reply:
Done